list

```diff
@@ -373,7 +373,6 @@ dtsx.io
 dub.sh
 dv.gd
 dvrv.ai
-dw.com
```

Comment:
Thanks for your suggestion, I believe that it's added because of the links as below, 
- https://dw.com/p/4Vb6v ➡️ https://www.dw.com/en/brics-expansion-united-states-germany-reactions/a-66636169 
- https://dw.com/p/4TSil ➡️ https://www.dw.com/en/how-is-europes-far-right-capitalizing-on-frances-unrest/a-66127263
- https://dw.com/p/4SqUY ➡️ https://www.dw.com/en/what-caused-the-implosion-of-the-titan-sub/a-65980310

Just they shared the exactly same domain, right?

Reply:
Yes, that happens, not separate domain or subdomain, but a folder. In that cases it should be URL rules, but not hosts. Of course, this solution is for uBlock/Adblock, and not for DNS-related blocking
||dw.com/p/

Reply:
Just curious, so you use this list as blocker, and dw.com was falsely blocked then?